packages/playground/src/pages/Components/Components.js

```diff
@@ -97,10 +96,13 @@ getPaymentMethods({ amount, shopperLocale }).then(async paymentMethodsResponse =
     window.upi = checkout.create('upi').mount('.upi-field');
 
     // PIX
-    window.pix = checkout.create('pix', { countdownTime: 5 }).mount('.pix-field');
+    window.pix = checkout.create('pix', { countdownTime: 5, showPayButton: true }).mount('.pix-field');
 
     // Oxxo
-    window.oxxo = checkout.create('oxxo').mount('.oxxo-field');
+    window.oxxo = checkout.create('oxxo', { showPayButton: true }).mount('.oxxo-field');
+
```

Comment:
Do we need to set `showPayButton: true` here?

Reply:
This is purely so the Pix component in the components page in the playground isn't just an empty html element.
TBH I don't know why components like these are added to the components page